platformcomponents/desktop/button.json

```diff
@@ -6,7 +6,7 @@
       "#normal": {
         "background": "@theme-button-secondary-normal",
         "text": "@theme-text-primary-normal",
-        "border": "none"
+        "border": "@theme-button-border-normal"
```

Comment:
there is no border for ghost button for HC themes though, right?

Reply:
`theme-button-border-normal` is a transparent border for non-HC themes - there is no border for the `ghost` button, in non-HC themes.
For HC though, all the buttons have a border, including `ghost` - this can be visible in the link attached in the PR description, under the `Links` header.